test/unit/client/test_client.py

```diff
@@ -526,6 +578,103 @@ def test_create_enterprise_user_returns_the_correct_user_object(mock_client, moc
     assert new_user.name == test_user_name
 
 
+def test_webhook_initializer(mock_client):
+    expected_id = '1234'
+    webhook = mock_client.webhook(expected_id)
+    assert isinstance(webhook, Webhook)
+    assert webhook.object_id == expected_id
+
+
+def test_create_webhook_on_folder_returns_the_correct_policy_object(
+        test_folder,
+        test_webhook,
+        mock_client,
+        mock_box_session,
+        create_webhook_on_folder_response,
+):
+    # pylint:disable=redefined-outer-name
+    expected_url = "{0}/webhooks".format(API.BASE_API_URL)
+    expected_body = {
+        'target': {
+            'type': test_folder.object_type,
+            'id': test_folder.object_id,
+        },
+        'triggers': ['FILE.UPLOADED', 'FOLDER.CREATED'],
+        'address': 'https://test.com',
+    }
+    value = json.dumps(expected_body)
+    mock_box_session.post.return_value = create_webhook_on_folder_response
+    new_webhook = mock_client.create_webhook(test_folder, ['FILE.UPLOADED', 'FOLDER.CREATED'], 'https://test.com')
+    mock_box_session.post.assert_called_once_with(
+        expected_url,
+        data=value,
+    )
+    assert isinstance(new_webhook, Webhook)
+    assert new_webhook.id == test_webhook.object_id
+    assert new_webhook.type == test_webhook.object_type
+    assert new_webhook.target['type'] == test_folder.object_type
+    assert new_webhook.target['id'] == test_folder.object_id
+    assert new_webhook.triggers == ['FILE.UPLOADED', 'FOLDER.CREATED']
+    assert new_webhook.address == 'https://test.com'
+
+
+def test_create_webhook_on_file_returns_the_correct_policy_object(
```

Comment:
Rather than duplicating this test, it might be better to create a fixture that enumerates both the file and folder fixtures — see the `test_item_and_response` fixture in test_item.py